libs/metamask/src/lib/+state/erc1155.ts

```diff
@@ -11,59 +11,53 @@ interface ERC1155_Event {
   URI: (value: string, id: number) => void;
 }
 
+export interface ERC1155_Meta {
+  name: string;
+  description: string;
+  image: string;
+  image_data?: string;
+  background_color?: string;
+  animation_url?: string;
+  youtube_url?: string;
+  attributes: { display_type?: string, trait_type: string, value: string }[];
```

Comment:
variables with `_` instead of camel case. Is it because of openSea specs ?

Reply:
yes